architecture_diagrams/excalidraw_links

```diff
@@ -0,0 +1,11 @@
+# Links to editable versions of the diagrams
```

Comment:
Nice, haven't seen this tool but I love that they're immediately editable without someone being able to compromise the linked one 👍